hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/Federation.md

```diff
@@ -78,7 +78,7 @@ to minimize overhead on the scheduling infrastructure (more in section on scalab
 3. Intercepts all the requests, thus it can enforce application quotas, which would not be enforceable by sub-cluster RM (as each only see a fraction of the AM requests).
 4. The AMRMProxy can enforce load-balancing / overflow policies.
 
-###Global Policy Generator
```

Comment:
In the markdown document, the title should have spaces, so I modified this place.